test/models/role_test.rb

```diff
@@ -209,7 +209,8 @@ def test_allow_to_modify_roles_when_bypass_locking
 
     context "when current user is admin for_current_user should return all givable roles" do
       test "Admin user should query Role model with no restrictions" do
-        assert_include Role.for_current_user.to_sql, 'WHERE "roles"."builtin" = 0'
```

Comment:
we talked with @xprazak2 it would be nicer to verify the method behavior on loading objects with builtin 0 and 1 rather then checking what SQL it generates